src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -128,7 +129,13 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                encodedAuthorization = "Basic "+new String(Base64.encodeBase64(authorization.getBytes()));
+                final Charset charset;
+                try {
+                    charset = Charset.forName("UTF-8");
```

Comment:
No, what I mean is, there is a `java.nio.charset.StandardCharsets`, but if the minimum supported version is 5 (like it says in the `pom.xml` then don't use it.
Then outlining it to a static method sounds better.